src/Visualization.jl

```diff
@@ -1,23 +1,46 @@
 using Graphs: SimpleGraph, Edge, nv
-using GraphMakie: graphplot!, GraphPlot
+using GraphMakie: graphplot!, GraphPlot, to_colormap, get_node_plot
 using Combinatorics: combinations
+using GraphMakie.NetworkLayout: IterativeLayout
 import Makie
 
-Makie.plottype(::TensorNetwork) = GraphPlot
+const MAX_NODE_SIZE = 20.
 
-function Makie.plot!(P::GraphPlot{Tuple{TensorNetwork{A}}}; kwargs...) where {A<:Ansatz}
-    tn = P[1][]
+function Makie.plot(tn::TensorNetwork{A}; kwargs...) where {A<:Ansatz}
+    f = Makie.Figure()
 
-    pos = IdDict(tensor => i for (i, tensor) in enumerate(tensors(tn)))
-    graph = SimpleGraph([Edge(pos[a], pos[b]) for ind in inds(tn) for (a, b) in combinations(links(ind), 2)])
+    p, ax = Makie.plot!(f[1,1], tn; kwargs...)
+    display(f)
+
+    return f, ax, p
+end
 
-    scene = Makie.parent_scene(P)
+function Makie.plot!(f::Makie.GridPosition, tn::TensorNetwork{A}; kwargs...) where {A<:Ansatz}
+    scene = Makie.Scene()
     default_attrs = Makie.default_theme(scene, GraphPlot)
 
     kwargs = Dict{Symbol,Any}(kwargs)
-    if P.attributes.node_size[] == default_attrs.node_size[]
-        kwargs[:node_size] = [max(10, log2(size(tensors(tn, i)) |> prod)) for i in 1:nv(graph)]
+
+    pos = IdDict(tensor => i for (i, tensor) in enumerate(tensors(tn)))
+    graph = SimpleGraph([Edge(pos[a], pos[b]) for ind in inds(tn) for (a, b) in combinations(links(ind), 2)])
+
+    log_size = [log2(size(tensors(tn, i)) |> prod) for i in 1:nv(graph)]
+
+    min_size, max_size = extrema(log_size)
+
+    kwargs[:node_size] = (log_size/max_size) * MAX_NODE_SIZE
```

Comment:
I was quit satisfied with the previous formula. Node size should not be normalized to min-max sizes of tensors. If a node is small, it's small because the tensor is small and the intuition should work that way.

```suggestion
    kwargs[:node_size] = [max(10, log2(size(tensors(tn,i)) |> prod)) for i in 1:nv(graph)]
```